python2/artifactfile.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "https://raw.githubusercontent.com/Azure/azure-devtestlab/master/schemas/2015-01-01/dtlArtifacts.json",
+  "title": "Python 2.7",
```

Comment:
Looks like python2 will install the latest 2.x version available. Right now that is 2.7 but may change soon. How about setting the title to the same as listed in Chocolatey: Python 2.x.